ide/vscode/launch-remote-openocd.json

```diff
@@ -13,7 +13,7 @@
             // Connect to an already running OpenOCD instance
             "gdbTarget": "your-openocd:3333",
             "svdFile": "${env:PICO_SDK_PATH}/src/rp2040/hardware_regs/rp2040.svd",
-            "runToMain": true,
+            "runToEntryPoint": "Main",
```

Comment:
Should this be "main" with a lowercase 'm', to match the other .json files here? :man_shrugging:

Reply:
Good spot! Resolved [0153dafc](https://github.com/mjcross/pico-examples/commit/0153dafc773f6d3006fb85baa3f7c95a04f1484c)